libcxx/docs/Status/FormatPaper.csv

```diff
@@ -4,7 +4,7 @@ Section,Description,Dependencies,Assignee,Status,First released version
 `[time.syn] <https://wg21.link/time.syn>`_,"Formatter ``chrono::sys_time<Duration>``",,Mark de Wever,|Complete|,17
 `[time.syn] <https://wg21.link/time.syn>`_,"Formatter ``chrono::utc_time<Duration>``",A ``<chrono>`` implementation,Mark de Wever,|Complete|,20
 `[time.syn] <https://wg21.link/time.syn>`_,"Formatter ``chrono::tai_time<Duration>``",,Mark de Wever,|Complete|,21
-`[time.syn] <https://wg21.link/time.syn>`_,"Formatter ``chrono::gps_time<Duration>``",A ``<chrono>`` implementation,Mark de Wever,,,
+`[time.syn] <https://wg21.link/time.syn>`_,"Formatter ``chrono::gps_time<Duration>``",,Mark de Wever,|Complete|,21
```

Comment:
https://github.com/llvm/llvm-project/blob/83fa117f76f9c4c82ce0ca914c4eba268c6c2fa2/libcxx/docs/Status/Cxx20Papers.csv#L118
https://github.com/llvm/llvm-project/blob/83fa117f76f9c4c82ce0ca914c4eba268c6c2fa2/libcxx/docs/Status/Cxx20Issues.csv#L272

Let's also change these lines, `Closes #100432`, `Closes #100014`, and update `libcxx/docs/ReleaseNotes/21.rst`.

Reply:
good point! I knew I still had to do that, but then forgot it.